client/src/components/footer/style.css

```diff
@@ -0,0 +1,19 @@
+footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 70px;
+  background: #d65050;
+}
+
+.footer--a {
+  color: white;
+  font-size: 35px;
+  height: 50px;
+  margin: 0 10px;
+  cursor: pointer;
+}
+.footer--a:hover {
```

Comment:
The color is already white so there is no need for the hover rule.

Reply:
there is hover rule to blue from ant design just I don't need the blue